src/components/DeploymentDetails.vue

```diff
@@ -6,9 +6,15 @@
       </template>
     </p-key-value>
 
-    <p-key-value label="Work Queue" :alternate="alternate">
-      <template v-if="deployment.workQueueName" #value>
-        <WorkQueueIconText :work-queue-name="deployment.workQueueName!" />
+    <p-key-value v-if="deployment.workPoolName" label="Work Pool" :alternate="alternate">
+      <template #value>
+        <WorkPoolIconText :work-pool-name="deployment.workPoolName!" />
+      </template>
+    </p-key-value>
+
+    <p-key-value v-if="deployment.workQueueName" label="Work Queue" :alternate="alternate">
+      <template #value>
+        <WorkQueueIconText :work-queue-name="deployment.workQueueName!" :work-pool-name="deployment.workPoolName" />
```

Comment:
Is it possible that both work queue name and work pool name are set? If so, should we show both? I'm not sure I understand the implications of that scenario

Reply:
I don't understand the question

Reply:
I added both to UI. If we have one of the set, we show one of them, if we have both we show both, if none - none
<img width="349" alt="Screenshot 2023-02-08 at 1 49 19 PM" src="https://user-images.githubusercontent.com/40467112/217624257-f8823ed8-2f39-4d46-84f1-509d25a9b7ab.png">

Reply:
i think i misunderstood the queue changes here, you can ignore this